packages/frontend/src/components/rec_profile.tsx

```diff
@@ -0,0 +1,65 @@
+import { calculateAverageRating } from "@esp-group-one/types";
+import type { CensoredUser } from "@esp-group-one/types";
+import moment from "moment";
+import { ProfilePic } from "./profile_pic";
+import { Stars } from "./stars";
+
+interface Info {
+  user: CensoredUser;
+  pfp: string;
+  availability: string[];
```

Comment:
Could we specify a `Moment` time or the `DateTimeString` type instead of just having it has `string`